### PR TITLE
fix(learn): Remove unnecessary inline code tag in challenge

### PR DESCRIPTION
**Pull request by kelvinsanchez15**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] All the files I changed are in the same world language, for example: only English changes, or only Chinese changes, etc.

To keep consistency I followed the sentence structure of the other comparison operator challenges. 

There is one challenge that also needs a change, it is worded like this: "Like the equality operator, the `>=` will convert data types while comparing." But that line comes from a review before the audit was merged. I can add the changes here depending on the path taken.

Challenge: [Comparison with the Greater Than Or Equal To Operator](https://github.com/freeCodeCamp/freeCodeCamp/blob/main/curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-greater-than-or-equal-to-operator.md)

### CODE REVIEW

File: curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-or-equal-to-operator.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ dashedName: comparison-with-the-less-than-or-equal-to-operator
 
 # --description--
 
-The less than or equal to operator (`<=`) compares the values of two numbers. If the number to the left is less than or equal to the number to the right, it returns `true`. If the number on the left is greater than the number on the right, it returns `false`. Like the equality operator, `less than or equal to` converts data types.
+The less than or equal to operator (`<=`) compares the values of two numbers. If the number to the left is less than or equal to the number to the right, it returns `true`. If the number on the left is greater than the number on the right, it returns `false`. Like the equality operator, less than or equal to operator converts data types.
```
Comment (RandellDawson) [inline curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-javascript/comparison-with-the-less-than-or-equal-to-operator.md:12]:
```suggestion
The less than or equal to operator (`<=`) compares the values of two numbers. If the number to the left is less than or equal to the number to the right, it returns `true`. If the number on the left is greater than the number on the right, it returns `false`. Like the equality operator, the less than or equal to operator converts data types.
```

**Review by RandellDawson — CHANGES_REQUESTED** [review]
Let's add the word "the" here.

**Comment by kelvinsanchez15** [comment]
@RandellDawson I added changes to the other similar challenges to keep it consistent.

**Review by RandellDawson — APPROVED** [review]

**Review by naomi-lgbt — APPROVED** [review]
Looks good to me.

**Pull request merged**